errata_tool/erratum.py

```diff
@@ -45,6 +45,7 @@ def _do_init(self):
         self.text_only = False
         self.text_only_cpe = None
         self.publish_date_override = None
+        self.publish_date = None
```

Comment:
Let's add a unit test to cover this new attribute. In `errata_tool/tests/test_advisory.py`, you can add a new method like this:

```python
    def test_publish_date(self, advisory):
        assert advisory.publish_date == '2018-Sep-26'
```

Reply:
Done